src/libs/antares/study/include/antares/study/area/links.h

```diff
@@ -21,6 +21,7 @@
 #ifndef __ANTARES_LIBS_STUDY_LINKS_H__
 #define __ANTARES_LIBS_STUDY_LINKS_H__
 
+#include <antares/study/parts/thermal/cluster.h>
```

Comment:
This will create circular dependency between the generator and the data structures.
In the future we want a separate tool that's depends only on structures and specialize what's needed after.

Reply:
If we move ThermalLaw to ts-generator.h,

ts-generator depends on ThermalCluster
ts-generator depends on AreaLink

That's all, right ?